auth-web/src/registerServiceWorker.ts

```diff
@@ -1,53 +1,15 @@
-/* eslint-disable no-console */
-
-import ConfigHelper from './util/config-helper'
-import { SessionStorageKeys } from './util/constants'
 import { register } from 'register-service-worker'
 
 if (process.env.NODE_ENV === 'production') {
   register(`${process.env.BASE_URL}service-worker.js`, {
-    ready () {
-      console.log(
-        'App is being served from cache by a service worker.\n' +
-        'For more details, visit https://goo.gl/AFskqB'
-      )
-    },
-    registered (event) {
-      console.log('Service worker has been registered.')
-    },
-    cached () {
-      console.log('Content has been cached for offline use.')
-    },
-    updatefound () {
-      console.log('New content is downloading.')
-    },
-    updated (updatedEvent) {
-      // write code for popup here
-      console.log('New content is available; refreshing page in 1 sec')
+    updated () {
       // remove older cached content
       // ref: https://santhoshkumarravi.medium.com/vue-pwa-disable-5463e44b1f7f
       caches.keys().then(names => {
```

Comment:
This gets uninstalled after running, and never needs to run again.

Reply:
![image](https://user-images.githubusercontent.com/3484109/177482010-7ce78677-69ff-4a01-a3b1-d1f441f3765c.png)